packages/zitadel-react/src/components/ZitadelReactProvider.tsx

```diff
@@ -0,0 +1,10 @@
+export type ZitadelReactProps = {
+  dark: boolean;
+  children: React.ReactNode;
+};
+
+export function ZitadelReactProvider({ dark, children }: ZitadelReactProps) {
+  return (
+    <div className={`${dark ? "ztdl-dark" : "ztdl-light"} `}>{children}</div>
+  );
+}
```

Comment:
Why not just ZitadelProvider and ZitadelProps?

Reply:
I thought its more suitable to not have conflicts with multiple packages later